### ISSUE TITLE
Subs not appearing for the second episode on a row

### ISSUE THREAD

**Issue by marouamghar** [jellyfin/jellyfin-ios#420]
**Describe the bug**
When watching two episodes on a row, subtitles don't appear on the second one. I need to go back to the lib and select the video for subs to appear correctly.

**System (please complete the following information):**
 - iOS Version: 15.7.3
 - Jellyfin Expo version: 43.0.0
 - Jellyfin Server version: 10.8.9

**Comment by thornbill** [comment]
Is this issue still present on the latest server version?

**Comment by marouamghar** [comment]
No, I don't think so. I don't know when/how/if it got fixed. I'll close it for now.
